unified-runtime/source/adapters/opencl/adapter.hpp

```diff
@@ -7,18 +7,22 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
+#pragma once
```

Comment:
I think it's worth doing ifdefs rather than pragma once, consensus online does seem to be that ifdefs are better

Reply:
Other files in this directory use `#pragma once`, so I chose it for consistency.